allennlp/models/semantic_parsing/wikitables/wikitables_semantic_parser.py

```diff
@@ -161,7 +149,16 @@ def _get_initial_state_and_scores(self,
                                       world: List[WikiTablesWorld],
                                       actions: List[List[ProductionRuleArray]],
                                       example_lisp_string: List[str] = None,
-                                      add_world_to_initial_state: bool = False) -> Dict:
+                                      add_world_to_initial_state: bool = False,
+                                      terminal_actions: List[torch.LongTensor] = None,
```

Comment:
This could also just be a `ChecklistState` object, I think.

Reply:
Done.